### ISSUE TITLE
Add The Adventures of Captain Proton

### ISSUE THREAD

**Issue by mariozepedap** [lynnandtonic/nestflix.fun#122]
From Star Trek Voyager

**Comment by HarleysTwin** [comment]
@lynnandtonic Technically this is a holonovel from _Star Trek: Voyager_.  Would that count as being a "movie," since movies don't seem to exist in the Star Trek universe?

**Comment by lynnandtonic** [comment]
Yeah I wasn't sure on this one either. But tons of people have submitted it so I will probably include it.

**Comment by lynnandtonic** [comment]
Ok I actually decided not to include holonovels or other holodeck simulation type things. Sorry, and thank you!
